test/e2e/catalog_e2e_test.go

```diff
@@ -178,18 +184,19 @@ var _ = Describe("Catalog", func() {
 
 		// Get updated catalogsource
 		fetchedUpdatedCatalog, err := fetchCatalogSourceOnStatus(crc, mainCatalogName, globalNS, catalogSourceRegistryPodSynced)
-		require.NoError(GinkgoT(), err)
+		Expect(err).ShouldNot(HaveOccurred())
 
 		subscription, err = fetchSubscription(crc, testNamespace, subscriptionName, subscriptionStateUpgradePendingChecker)
-		require.NoError(GinkgoT(), err)
-		require.NotNil(GinkgoT(), subscription)
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(subscription).ShouldNot(BeNil())
 
 		// Ensure the timing
 		catalogConnState := fetchedUpdatedCatalog.Status.GRPCConnectionState
 		subUpdatedTime := subscription.Status.LastUpdated
 		timeLapse := subUpdatedTime.Sub(catalogConnState.LastConnectTime.Time).Seconds()
-		require.True(GinkgoT(), timeLapse < 60)
+		Expect(timeLapse).Should(BeNumerically("<", 60))
```

Comment:
But, I don't see if it could be used here since we are asserting that the var `timeLapse` has a value < 60. `BeTemporarily` expects 60 to be of type `time.Time{}`

Reply:
I think Ben is suggesting something like `Expect(catalogConnState.LastConnectTime.Time).Should(BeTemporally("<", subUpdatedTime.Add(60 * time.Second)` @harishsurf

Reply:
changed it to: 
`Expect(subUpdatedTime.Time).Should(BeTemporally("<", catalogConnState.LastConnectTime.Add(60*time.Second)))`